### PR TITLE
Updated LinkedIn link

### PR DESCRIPTION
**Pull request by aindree-2005**
## What does this PR do?

This PR has added LinkedIn link of Appwrite to our ReadMe. This will allow open source contributors to follow AppWrite on LinkedIn and be in touch with job openings and other events taking place
## Test Plan
I have not made coding changes, only updated ReadMe file

## Related PRs and Issues

A prior unfixed issue of adding LinkedIn

## Checklist

- [ Y] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [Y ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ For security issues, kindly email us at [security@appwrite.io](mailto:security@a
 
 ## Follow Us
 
-Join our growing community around the world! Check out our official [Blog](https://medium.com/appwrite-io). Follow us on [Twitter](https://twitter.com/appwrite), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/), [Dev Community](https://dev.to/appwrite) or join our live [Discord server](https://discord.gg/GSeTUeA) for more help, ideas, and discussions.
+Join our growing community around the world! Check out our official [Blog](https://medium.com/appwrite-io). Follow us on [Twitter](https://twitter.com/appwrite), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/), [Dev Community](https://dev.to/appwrite),[LinkedIn](https://www.linkedin.com/company/appwrite/?originalSubdomain=il) or join our live [Discord server](https://discord.gg/GSeTUeA) for more help, ideas, and discussions.
```
Comment (Susmita-Dey) [inline README.md:216]: Change linkedin link:- https://www.linkedin.com/company/appwrite/

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Kindly change the old logo that you're trying to push as we are having our new brand logo.

File: README.md (hunk 1 above)
Comment (aindree-2005) [reply README.md]: > Change linkedin link:- https://www.linkedin.com/company/appwrite/

Changed the link! Ok, I'll remove the logo file

**Comment by stnguyen90** [comment]
Unfortunately, we don't want to add this at the moment. If you wish to contribute as part of Hacktoberfest, please look for issues labeled "hacktoberfest".

**Pull request closed without merging**